projects/storefrontlib/src/cms-components/myaccount/order/cancellations-returns/return-order-confirmation/return-order-confirmation.component.ts

```diff
@@ -105,6 +105,13 @@ export class ReturnOrderConfirmationComponent implements OnDestroy {
     }
   }
 
+  back(): void {
```

Comment:
I notice when you click back the item counters are all reset to 0. Would it be expected for the value to still be populated?

Reply:
I created a ticket #5583 for it, because I need think more about where to keep the entry inputs. And, I don't want it block styling.